WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -56,6 +56,8 @@ class StoreStatsV4PeriodViewController: UIViewController {
     @IBOutlet private weak var chartAccessibilityView: UIView!
     @IBOutlet private weak var noRevenueView: UIView!
     @IBOutlet private weak var noRevenueLabel: UILabel!
+    @IBOutlet private weak var timeRangeBarView: StatsTimeRangeBarView!
```

Comment:
Since this is in the .xib file and not code-generated, it will need autolayout constraints defined. There are several missing constraints for different views:

<img width="814" alt="Screen Shot 2019-08-16 at 9 54 52 AM" src="https://user-images.githubusercontent.com/1062444/63176902-6cdbde80-c00c-11e9-8951-6a42a08901d1.png">

When transitioning from landscape to portrait, the layout shifts:
![Simulator Screen Shot - iPhone Xs - 2019-08-16 at 09 59 37](https://user-images.githubusercontent.com/1062444/63177014-a9a7d580-c00c-11e9-8c36-b27d33b0ea4e.png)

<img src="https://user-images.githubusercontent.com/1062444/63177059-bfb59600-c00c-11e9-998f-d712cd4b96dc.png" width = "350" />

Reply:
> Since this is in the .xib file and not code-generated, it will need autolayout constraints defined. There are several missing constraints for different views:

I believe the errors about Auto Layout constraints are from the xib not knowing the content of the custom view class `StatsTimeRangeBarView` since this view is defined in the code, not in Interface Builder. Normally, a `UIStackView`'s arranged subviews shouldn't need to specify the position along the stack view axis or height, given each subview has their height defined within their own and if the stack view's `alignment` and `distribution` are set to `fill`. I looked online, it looks like [a way to calm the xib Auto Layout checker](https://stackoverflow.com/questions/40933819/y-constraint-missing-in-vertical-stackview) is to add a height constraint at a lower priority and remove at build time.

Reply:
> When transitioning from landscape to portrait, the layout shifts:

Reply:
> When transitioning from landscape to portrait, the layout shifts:

do you have steps to reproduce this issue? I tested on iPhone 5s simulator and iPhone 6s Plus device, and wasn't able to repro this scroll view offset issue with a bunch of rotations

Reply:
Sure. With an iPhone XS simulator, run the app. When the My store tab is loaded, use Cmd + ➞ to rotate the screen, so the notch is on the righthand side. Then Cmd + ↑ to return to portrait mode. 

Here's a [demo movie](https://cld.wthms.co/h1uReN)...the animated .gif was too large to post here. I pulled the latest changes and can still reproduce this.

Reply:
> it looks like a way to calm the xib Auto Layout checker is to add a height constraint at a lower priority and remove at build time

lol. Works like a charm. Thank you!